composeApp/src/commonMain/kotlin/com/rahulrav/camera/scan/SupportedAlphaCamera.kt

```diff
@@ -0,0 +1,29 @@
+package com.rahulrav.camera.scan
+
+enum class SupportedAlphaCamera(
+    val modelCode: String,
+    val modelName: String,
+    val modelNumber: String
+) {
+  ILCE_7M4(
+      modelCode = "TODO",
+      modelNumber = "ILCE-7M4",
+      modelName = "Sony Alpha 7 IV",
+  ),
+  ILCE_7RM5(
+      modelCode = "E1",
```

Comment:
From what I can tell, Sony uses `E1` as the model code in the manufacturer specific data for all their cameras. The protocol version is subtly different FWICT, its either `64` or `65`; but I have not been able to identify the differences.

For the user friendly name, that is readily available as a separate field on the Advertisement, and that is something we can rely on to be more graceful against cameras' we don't really test against.

Reply:
I'll drop this then. I suppose there is no way to know the actual model if the user changes the name, for compat checks reasons, then?

Reply:
I don't think there is. I think as long as it's a Sony Camera + has the right BLE characteristics we should be okay. We can gracefully handle the cases we don't find the expected services.